packer/packer_source.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+ERR_PACKER_COPY_FILE=113 # Error writing a file to disk during VHD CI
```

Comment:
I vote we keep all error code definitions in the same file to avoid spreading information